All_home_works/hw7/task1.py

```diff
@@ -0,0 +1,34 @@
+"""
+Given a dictionary (tree), that can contains multiple nested structures.
+Write a function, that takes element and finds the number of occurrences
+of this element in the tree.
+Tree can only contains basic structures like:
+    str, list, tuple, dict, set, int, bool
+"""
+from typing import Any
+
+
+def find_occurrences(tree: dict, element: Any) -> int:
+    counter = 0
+    if tree == element and type(tree) == type(element):
+        counter += 1
+    elif isinstance(tree, tuple) or isinstance(tree, list) or isinstance(tree, set):
```

Comment:
We can simplify this line. Please check the documentation.

Reply:
`isinstance(tree, (list, tuple, set))`